Makefile

```diff
@@ -38,6 +38,6 @@ endif
 ifndef BEAME_TESTS_ROOT_CREDS_FQDN
 	$(error BEAME_TESTS_ROOT_CREDS_FQDN is undefined)
 endif
-	DEBUG="beame:sdk:unittests:*" BEAME_LOG_LEVEL=FATAL ./node_modules/mocha/bin/mocha tests/unit_tests/*.js
+	DEBUG="beame:sdk:unittests:*" BEAME_LOG_LEVEL=ERROR ./node_modules/mocha/bin/mocha tests/unit_tests/*.js
```

Comment:
GlobalSign cred error was being hidden when running unittests. Change made to prevent this issues from being swallowed